clang/lib/CIR/CodeGen/CIRGenExprCXX.cpp

```diff
@@ -876,11 +876,97 @@ void CIRGenFunction::emitNewArrayInitializer(
   unsigned InitListElements = 0;
 
   const Expr *Init = E->getInitializer();
+  QualType::DestructionKind DtorKind = ElementType.isDestructedType();
   CleanupDeactivationScope deactivation(*this);
+  bool pushedCleanup = false;
```

Comment:
This seems unused, if you are intending to just keep track of lacking support for cleanup, using `assert(!cir::MissingFeatures::requiresCleanups());` is one option.

Reply:
You're right, it is unused. The place where it would be used is covered by an "NYI" so I'll just remove it until it's needed.

Reply:
sounds good